policyengine_us/parameters/gov/states/co/tax/income/credits/sales_tax_refund/amount/scale.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   reference:
     - title: Colo. Code Regs. 39-22-2003 State Sales Tax Refund (1), (b)
       href: https://casetext.com/regulation/colorado-administrative-code/department-200-department-of-revenue/division-201-taxation-division/rule-1-ccr-201-2-income-tax/rule-39-22-2003-effective-until-8302023-state-sales-tax-refund
+    # In 2023 Colorado switched to a flat amount as opposed to a scale based on income.
+    - title: 2023 Colorado Individual Income Tax Filing Guide
+      href: https://leg.colorado.gov/sites/default/files/documents/2023A/bills/fn/2023a_hb1311_00.pdf#page=3
```

Comment:
Please add a note or source showing that this bill was passed and signed by the governor.